README.md

```diff
@@ -65,6 +65,36 @@ Web context path to use, such as "/rundeck". http://host.domain:port/server_web_
 #####`ssl_enabled`
 Enable ssl for the Rundeck web application.
 
+#####`ssl_keyfile`
+#####`ssl_certfile`
+If ssl enabled you need set up this. You need provide the crt and key files in your profile oder role manifest.
+
+How to: eg: environments/role/manifests/rundeck.pp
```

Comment:
Drop this line and include the sample file path it represents as a comment inside the appropriate code block.